source/stylesheets/modules/_anchored-heading.scss

```diff
@@ -1,14 +1,37 @@
+$anchored-heading-icon-width: 23px;
```

Comment:
I don't know if scss scope works like less scope, but given these are only ever used within `.anchored-heading__icon`, could these be defined inside that selector rather than at the global / file level? Could potentially also then make them slightly less verbose as the context would be clear.